kitsune/search/v2/base.py

```diff
@@ -43,6 +43,10 @@ def prepare(cls, instance):
 
         return obj
 
+    @classmethod
+    def get_queryset(cls):
+        return cls.get_model()._default_manager
```

Comment:
I am curious about this, pls tell me more

Reply:
This is essentially calling `cls.get_model().objects`, so if we created a `FoobarDoc(SumoDocument)` with `get_model()` returning `Foobar`, this would be returning `Foobar.objects`.

`_default_manager` seems to be the more "djangonic" way to refer to `.objects` in case the manager has been renamed: https://docs.djangoproject.com/en/2.2/topics/db/managers/#django.db.models.Model._default_manager

The reason for adding this as a method is to be able to control the queryset used when indexing from the document, so we can add optimisations and aggregations and so on.